firebase_admin/tenant_mgt.py

```diff
@@ -347,7 +365,9 @@ def list_tenants(self, page_token=None, max_results=_MAX_LIST_TENANTS_RESULTS):
         if page_token:
             payload['pageToken'] = page_token
         try:
-            return self.client.body('get', '/tenants', params=payload)
+            body = self.client.body('get', '/tenants', params=payload)
+            _auth_utils.convert_tenant_list_auth_payload_to_user(body)
+            return body
```

Comment:
Tenant(body)?

Reply:
This is a list of tenants, so I'm not sure if Tenant(body) applies here.